webapp/package-lock.json

```diff
@@ -3386,20 +3377,17 @@
         "@emotion/hash": {
           "version": "0.8.0",
           "resolved": "https://registry.npmjs.org/@emotion/hash/-/hash-0.8.0.tgz",
-          "integrity": "sha512-kBJtf7PH6aWwZ6fka3zQ0p6SBYzx4fl1LoZXE2RrnYST9Xljm7WfKJrU4g/Xr3Beg72MLrp1AWNUmuYJTL7Cow==",
-          "dev": true
+          "integrity": "sha512-kBJtf7PH6aWwZ6fka3zQ0p6SBYzx4fl1LoZXE2RrnYST9Xljm7WfKJrU4g/Xr3Beg72MLrp1AWNUmuYJTL7Cow=="
         },
         "@emotion/memoize": {
           "version": "0.7.4",
           "resolved": "https://registry.npmjs.org/@emotion/memoize/-/memoize-0.7.4.tgz",
-          "integrity": "sha512-Ja/Vfqe3HpuzRsG1oBtWTHk2PGZ7GR+2Vz5iYGelAw8dx32K0y7PjVuxK6z1nMpZOqAFsRUPCkK1YjJ56qJlgw==",
-          "dev": true
```

Comment:
I feel that many of the `integrity` and `dev: true` changes are not necessary. Do you know why these got introduced?

Reply:
My guess is the same problem we are having in some other places with npm versions. Do we have standarized what version shall we use to generate this files?

Reply:
@colorfusion What version of npm are you using?

Reply:
I'm currently using `6.14.8`, should I change to another version?

Reply:
@colorfusion I'm using `6.14.4` so I don't think that's the issue. Also, I'm seeing the same changes when I run this locally, so it must be the way that `react-select` is declaring its own dependencies. No problem here